article/sections/_2_graph_importance.tex

```diff
@@ -133,79 +119,22 @@ \section{Perils of disregard}
    \label{fig:SR3_causal_2}
 \end{figure}
 
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{WTW_interacting_graph}
-   \caption{Causal Graph for the Walk-Transit-Walk Utility Function}
-   \label{fig:WTW_causal_2}
-\end{figure}
-
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{WTD_interacting_graph}
-   \caption{Causal Graph for the Walk-Transit-Drive Utility Function}
-   \label{fig:WTD_causal_2}
-\end{figure}
-
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{DTW_interacting_graph}
-   \caption{Causal Graph for the Drive-Transit-Walk Utility Function}
-   \label{fig:DTW_causal_2}
-\end{figure}
-
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{WALK_interacting_graph}
-   \caption{Causal Graph for the Shared Ride 3+ Utility Function}
-   \label{fig:WALK_causal_2}
-\end{figure}
-
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{BIKE_interacting_graph}
-   \caption{Causal Graph for the Bike Utility Function}
-   \label{fig:BIKE_causal_2}
-\end{figure}
+To achieve this goal, we follow these simulation steps, using the same outcome model:
+\begin{itemize}
+   \item Simulate data from the DAGs shown in Figure \ref{fig:IND_GRAPH} through Figure \ref{fig:SR3_causal_2}.
```

Comment:
```suggestion
   \item Simulate data from the DAGs shown in Figure \ref{fig:DA_causal_2} through Figure \ref{fig:SR3_causal_2}.
```

Reply:
This should stay as IND_GRAPH because we simulate data based on the different causal graphs.

Reply:
Here's how I understand the simulation on observable's public notebook:
- From [L565](https://github.com/hassan-obeid/tr_b_causal_2020/blob/master/notebooks/final/7.0-mab-selection-on-observables-final.py#L565) - [L619](https://github.com/hassan-obeid/tr_b_causal_2020/blob/master/notebooks/final/7.0-mab-selection-on-observables-final.py#L619) you simulate data once as `long_sim_data`,
using the `SIMULATE_NODES_WIDE` script,
defined [here](https://github.com/hassan-obeid/tr_b_causal_2020/blob/master/notebooks/final/7.0-mab-selection-on-observables-final.py#L77) to reference [simworkflow.py](https://github.com/hassan-obeid/tr_b_causal_2020/blob/master/src/causal2020/observables/simworkflow.py),
which makes predictions based on `fig:DA_casuasl2`.
- `long_sim_data` is then copied to make `long_sim_data_naive` and `long_sim_data_causal` in [L647](https://github.com/hassan-obeid/tr_b_causal_2020/blob/master/notebooks/final/7.0-mab-selection-on-observables-final.py#L647).
These two dataframes are then further perturbed according to our do-operator / intervention.

I believe that what I've written above matches the description of "simulate from 1 graph, perturb 2 different ways, then see how the causal effect estimates turned out".
Let me know if and why (i.e. with links to code supporting why) you believe otherwise!

Reply:
@bouzaghrane Gentle ping on this.

Reply:
You are right. My apologies, I was just combining all of the simulation effort into one step in my mind.

Reply:
🙏🏽 No worries, thanks for making the change!